README.md

```diff
@@ -47,16 +47,18 @@ Themes working in **web-greeter** should work also here. All themes shipped with
 
 ### Common dependencies
 
-- lightdm (as obvious)
-- gobject-introspection
-- liblightdm-gobject
-- liblightdm-gobject-dev
-- libgirepository1.0-dev
-- libcairo2
-- libcairo2-dev
-- libxcb-dev
-- libx11-dev
-
+```sh
+sudo apt install \
+lightdm \
+gobject-introspection \
+liblightdm-gobject \
+liblightdm-gobject-dev \
+libgirepository1.0-dev \
+libcairo2 \
+libcairo2-dev \
+libxcb-dev \
+libx11-dev
+```
```

Comment:
I'd prefer to not use a specific package manager like `apt` as not everyone have a Debian distro. If you want to provide specific instructions for `apt`, it'd be better to provide the same for `dnf` and `pacman` (and maybe other package managers).